
<launch>
    <param name="use_sim_time" value="false"/>

    <!-- 设置当前机器人命名空间 -->
<!--    <arg name="namespace1" default="robot1"/>

    <node ns="$(arg namespace1)" name="agent_control_node" pkg="multi_agent_control" type="agent_control_node" output="screen" respawn="true">-->

<!--         <remap from="odom" to="/$(arg namespace1)/odom"/> -->
<!--         <remap from="cmd_vel" to="/robot1/cmd_vel"/> -->

<!--    <remap from="cmd_vel" to="cmd_vel"/>
        <param name="odom_frame_"  value="$(arg namespace1)_odom"/>
        <param name="base_frame_"  value="$(arg namespace1)_base_footprint"/>

    </node>-->
    
    
   
    <arg name="namespace4" default="robot4"/>

    <node ns="$(arg namespace4)" name="agent_control_node" pkg="multi_agent_control" type="agent_control_node" output="screen" respawn="true">
<!--        	<remap from="odom" to="/$(arg namespace4)/odom"/> -->
<!--         <remap from="cmd_vel" to="/robot4/cmd_vel"/> -->
        <remap from="cmd_vel" to="cmd_vel"/>
        <param name="odom_frame"  value="$(arg namespace4)_odom"/>
        <param name="base_frame"  value="$(arg namespace4)_base_footprint"/>
    </node>
</launch>
